src/types/Storage/methods/startWorker.spec.js

```diff
@@ -16,8 +16,10 @@ describe('Storage - startWorker', function suite() {
       autosaveIntervalTime: defaultIntervalValue,
     };
     startWorker.call(self);
-    expect(self.interval.constructor.name).to.be.equal('Timeout');
-    expect(self.interval._repeat).to.be.equal(defaultIntervalValue); // Timeout are null btw
+    if(!process.browser){
```

Comment:
Let's skip the whole test?

Reply:
We could, but why would we ? 
I mean, if we can test the setTimeout to always be a thing (in case of someone losing that unpurposely), why would we ? 
It does 99.9% of the job. Yes it doesn't test on browser, but it does on node.

Reply:
I meant:
```js
if (process.browser){
  this.skip('doesn\'t work in browser')
  return;
}
```

Reply:
Make total sense actually, there is nothing more after that except a clear of interval. 
Doing that !